services/bonita-time-tracker/src/test/java/org/bonitasoft/engine/tracking/RecordAssert.java

```diff
@@ -0,0 +1,121 @@
+package org.bonitasoft.engine.tracking;
```

Comment:
hum, I think it's better.
Usually, there are options to generate headers...
Do you modify some parts of the generated class?
Does this generated code will evolve regularly or not? 
If yes on one of this question, it might worth not committing the generated file.